### ISSUE TITLE
Default link should not have aria-label

### ISSUE THREAD

**Issue by cstrobbe** [ec-europa/europa-component-library#2845]
### Actual behaviour

The [Default link example](https://ec.europa.eu/component-library/ec/components/navigation/link/code/) has an `aria-label` attribute:

```
<p class="ecl-u-type-paragraph">The European Commission is the executive of <a href="/component-library/example"
    class="ecl-link" aria-label="Read more about the European Union">The European Union</a> and promotes its general
  interest.</p>
```

As a consequence, a screen reader will say,

> Read more about the European Union, link

This is because the `aria-label` attribute replaces the link text itself as the link's "accessible name".

Another consequence is that speech recognition users, who see the visual link "The European Union", will speek the visual link text, but the speech recognition will not be able to find an accessible name that begins with the words "The European Union". Hence, the speech recognition software won't activate that link. This is a violation of [WCAG success criterion 2.5.3 (Label in Name)](https://www.w3.org/TR/WCAG21/#label-in-name).

### Expected behaviour

The expected behaviour is that a screen reader announces that link as follows:

> The European Union, link

### Steps to reproduce the issue

See "Actual Behaviour", above.

### Recommendation

Do not generate an `aria-label` for normal text links. It is completely unnecessary for such links.

**Comment by planctus** [comment]
Thank you @cstrobbe, although our components are tested for their accessiblity both in code and in the demo you have in the playground of the website and they are further reviewed by an accessibility team, this issue was never raised and i agree with you that the usage we do of the aria-label for links in our demos is probably wrong, although it normally "contains" the label of the link, so from a speech recognition point of view this should still work, as you pointed out the experience with a text to speech software is not the correct one.
I will create a ticket on our board to handle this

**Comment by cstrobbe** [comment]
@planctus When you create that ticket, can you please ping me from there? (I don't know all the internal JIRA instances yet.)

**Comment by planctus** [comment]
this is the ticket url, https://citnet.tech.ec.europa.eu/CITnet/jira/browse/FRONT-3937 but you might not have access to it

**Comment by cstrobbe** [comment]
It turns out that I have access to it . Thanks :-)

**Comment by emeryro** [comment]
links have been updated. This is part of release 3.8.0
